src/js/actions/conversation-actions.js

```diff
@@ -54,3 +72,10 @@ export function resetUnreadCount() {
         type: RESET_UNREAD_COUNT
     };
 }
+
+export function setFetchingMoreMessagesFromServer(value) {
```

Comment:
This is to make sure there aren't multiple fetch requests happening at the same time
https://github.com/smooch/smooch-js/pull/374#discussion-diff-75224489